File: backend/src/main/java/com/ll/hotel/global/security/redis/RedisProperties.java
package com.ll.hotel.global.security.redis;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Data
@Component
@ConfigurationProperties(prefix = "custom.redis")
public class RedisProperties {

    private String host;
    private int port;
    private String password;
}

File: backend/src/main/java/com/ll/hotel/global/security/redis/RedisProperties.kt
package com.ll.hotel.global.security.redis

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties(prefix = "custom.redis")
data class RedisProperties(
    var host: String = "",
    var port: Int = 0,
    var password: String = ""
) 